AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4120,6 +4120,42 @@ function Debug-AzStorageAccountEntraKerbAuth {
                 Write-Error $_
             }
 
+        }
+        #
+        #Check if the machine is HAADJ or AADJ
+        #
+        if (!$filterIsPresent -or $Filter -match "CheckforHAADJ/AADJ")

```

Comment:
I think it's missing from the list of filters at the top. I would also call this something simpler, like `CheckDomainJoin`